CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(Boost REQUIRED)
+find_package(console_bridge_vendor REQUIRED)
 find_package(console_bridge REQUIRED)
```

Comment:
Do you need both, `console_bridge_vendor` and `console_bridge`?

Reply:
Hm, we can probably omit it, it's not exporting any targets.